src/metabase/server/instance.clj

```diff
@@ -0,0 +1,130 @@
+(ns metabase.server.instance
```

Comment:
I'm not seeing a deleted file this code came from - did you miss a `git rm`?

This seems like a lot of pretty core machinery not to have existed already.

Reply:
Oops, good catch. This is renamed from `metabase.server`. Fixed